<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>790</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pwd Management</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(85, 87, 83);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>771</width>
      <height>271</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout"/>
   </widget>
   <widget class="QPushButton" name="pushButton_save">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&amp;Save</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editDescription">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>271</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editPassword">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>50</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editKey">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>50</y>
      <width>181</width>
      <height>25</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>The default key to descript the password</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>271</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236)</string>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>ID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>251</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>20</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string>Key</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>45</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(85, 87, 83);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Search">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 209, 255)</string>
    </property>
    <property name="text">
     <string>S&amp;earch</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Update">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-color: rgb(196, 160, 0);*/</string>
    </property>
    <property name="text">
     <string>&amp;Update</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Editar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*background-color: rgb(206, 92, 0);*/</string>
    </property>
    <property name="text">
     <string>&amp;Edit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_New">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(78, 154, 6);</string>
    </property>
    <property name="text">
     <string>&amp;New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>&amp;Cancel</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>100</y>
      <width>201</width>
      <height>25</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(238, 238, 236);
background-color: rgb(46, 52, 54)</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>771</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <zorder>gridLayoutWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>lineEdit_search</zorder>
   <zorder>label_2</zorder>
   <zorder>spinBox</zorder>
   <zorder>pushButton_cancel</zorder>
   <zorder>pushButton_Editar</zorder>
   <zorder>label_4</zorder>
   <zorder>pushButton_Search</zorder>
   <zorder>pushButton_Update</zorder>
   <zorder>pushButton_New</zorder>
   <zorder>label_3</zorder>
   <zorder>label_5</zorder>
   <zorder>pushButton_save</zorder>
   <zorder>editDescription</zorder>
   <zorder>editPassword</zorder>
   <zorder>editKey</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>790</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="action_New"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionEditar">
   <property name="text">
    <string>&amp;Editar</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>spinBox</tabstop>
  <tabstop>editDescription</tabstop>
  <tabstop>editPassword</tabstop>
  <tabstop>editKey</tabstop>
  <tabstop>lineEdit_search</tabstop>
  <tabstop>pushButton_Search</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_save</tabstop>
  <tabstop>pushButton_Editar</tabstop>
  <tabstop>pushButton_Update</tabstop>
  <tabstop>pushButton_New</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_search</sender>
   <signal>returnPressed()</signal>
   <receiver>pushButton_Search</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>644</x>
     <y>195</y>
    </hint>
    <hint type="destinationlabel">
     <x>719</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
